graphdatascience/session/gds_sessions.py

```diff
@@ -70,7 +71,9 @@ def get_or_create(
         self,
         session_name: str,
         memory: SessionMemory,
+        # TODO support TTL and cloud_location
         db_connection: DbmsConnectionInfo,
+        cloud_location: Optional[CloudLocation] = None,
```

Comment:
lets add a notebook example (making sure its part of CI but also for user-visibility)

Reply:
-- will do in follow up PR